packages/cli/src/deps.rs

```diff
@@ -0,0 +1,191 @@
+use template::Template;
+
+use crate::colors::*;
+use crate::internal::template;
+use crate::package_manager::PackageManager;
+use std::process::Command;
+
+fn is_rustc_installed() -> bool {
+    Command::new("rustc").arg("-V").output().is_ok()
+}
+fn is_cargo_installed() -> bool {
+    Command::new("cargo").arg("-V").output().is_ok()
+}
+fn is_node_installed() -> bool {
+    Command::new("node").arg("-v").output().is_ok()
+}
+
+fn is_trunk_installed() -> bool {
+    Command::new("trunk").arg("-V").output().is_ok()
+}
+fn is_tauri_cli_installed() -> bool {
+    Command::new("cargo")
+        .arg("tauri")
+        .arg("-V")
+        .output()
+        .map(|o| {
+            let s = String::from_utf8_lossy(&o.stderr);
+            !s.starts_with("error:")
+        })
+        .unwrap_or(false)
+}
+fn is_wasm32_installed() -> bool {
+    Command::new("rustup")
+        .args(["target", "list", "--installed"])
+        .output()
+        .map(|o| {
+            let s = String::from_utf8_lossy(&o.stdout);
+            s.contains("wasm32-unknown-unknown")
+        })
+        .unwrap_or(false)
+}
+// fn is_pnpm_installed() -> bool {
+//     Command::new("pnpm").arg("-v").output().is_ok()
+// }
+// fn is_yarn_installed() -> bool {
+//     Command::new("yarn").arg("-v").output().is_ok()
+// }
+// fn is_npm_installed() -> bool {
+//     Command::new("npm").arg("-v").output().is_ok()
+// }
+// fn is_fnm_installed() -> bool {
+//     Command::new("fnm").arg("-V").output().is_ok()
+// }
+// fn is_nvm_installed() -> bool {
+//     #[cfg(windows)]
+//     return Command::new("nvm").arg("version").output().is_ok();
+//     #[cfg(not(windows))]
+//     Command::new("nvm").arg("-v").output().is_ok()
+// }
+
+pub fn print_missing_deps(pkg_manager: PackageManager, template: Template, alpha: bool) {
+    let rustc_installed = is_rustc_installed();
+    let cargo_installed = is_cargo_installed();
+    let deps: &[(&str, String, &dyn Fn() -> bool, bool)] = &[
+        (
+            "Rust",
+            format!("Visit {BLUE}https://www.rust-lang.org/learn/get-started#installing-rust{RESET}"),
```

Comment:
Making the phrasing the same as the rustc and cargo lines
```suggestion
            format!("Visit {BLUE}https://www.rust-lang.org/learn/get-started#installing-rust{RESET} to install Rust"),
```

Reply:
seems redundant, doesn't it? since we already informed the user the rust is the one missing

Reply:
Maybe just "to install" then? The perfectionist in me likes them to be the same phrase but that could just be me